eodatasets3/verify.py

```diff
@@ -39,8 +58,15 @@ def calculate_file_hash(filename, hash_fn=hashlib.sha1, block_size=4096):
     :return: String of hex characters.
     :rtype: str
     """
-    with Path(filename).open("rb") as f:
+    if is_uri(str(filename)):
+        bucket, key = get_bucket_key(filename)
+        s3client = boto3.client('s3', region_name='us-east-1')
```

Comment:
We should take the new region environment-variable here too

Reply:
I missed this one last time, done.